src/util/StringUtils.h

```diff
@@ -508,26 +508,30 @@ J join(const vector<J>& to_join, const S& joiner) {
 
 // _____________________________________________________________________________
 inline string lstrip(const string& text, char c) {
-  size_t i = 0;
-  while (i < text.size() && text[i] == c) {
```

Comment:
@niklas88 Actually you could probably template this to "everything that has an overload of find_first_not_of, so
`template <class S>`
`inline string lstrip(const string& text, S&& s)`
 `{ return text.substr(text.find_first_not_of(std::forward<S>(s))); }`

Reply:
Sadly templating `lstrip()` on the first argument doesn't work because then in `lstrip("foo", ' ')` the first  argument is deduced to `char[size]` which doesn't have `last_index_not_of()`

Reply:
That is exactly what I suggested.
I don't know if string_views have these methods, but const string&'s are sufficiently effective for now  I would guess.